A way to have additional addons repositories configured (#9013)
kyma-project/kyma: docs/helm-broker/05-01-helm-broker.md

| **ctrl.tmpDirSizeLimit** | Specifies a size limit on the `tmp` directory in the Helm Pod. This directory is used to store processed addons. Eviction manager monitors the disk space used by the Pod and evicts it when the usage exceeds the limit. Then, the Pod is marked as `Evicted`. The limit is enforced with a time delay, usually about 10s. | `1Gi` |
| **global.cfgReposUrlName** | Specifies the name of the default ConfigMap which provides the URLs of addons repositories. | `helm-repos-urls` |
| **global.isDevelopMode** | Defines whether to accept URL prefixes from the **global.urlRepoPrefixes.additionalDevelopMode** list. If set to `true`, the Helm Broker accepts the prefixes from the list. | `false` |
| **global.urlRepoPrefixes.default** | Defines a list of accepted prefixes for repository URLs. | `'https://', 'git::', 'github.com/', 'bitbucket.org/'` |
| **global.urlRepoPrefixes.additionalDevelopMode** | Defines a list of accepted prefixes for repository URLs when develop mode is enabled. | `'http://'` |

> | **additionalAddonsRepositories.myRepo** | A map of additional ClusterAddonsConfiguration repositories to create by default. The key gets used as name while the value gets used as URL for the repository. | `github.com/myOrg/myRepo//addons/index.yaml` |

Reviewer: ```suggestion
| **additionalAddonsRepositories.myRepo** | Provides a map of additional ClusterAddonsConfiguration repositories to create by default. The key is used as a name and the value is used as a URL for the repository. | `github.com/myOrg/myRepo//addons/index.yaml` |
```
